### PR TITLE
Improve template with additional packages

### PR DESCRIPTION
**Pull request by anitvam**
Changes:
- Removed non-working `image` shortcode
    - Update slides template with hugo's `figure` shortcode
- Add `multi-col` and `col` shortcodes
- Add working `font-awesome` dependency
- Add `Bootstrap` libraries
- Add `graph` shortcode leveraging `Graph.js` library
- Add `tick` and `cross` shortcodes leveraging font-awesome

### CODE REVIEW

File: assets/custom-theme.scss [hunk 1]
```diff
@@ -207,3 +207,23 @@ kbd {
   font-weight: normal;
 }
 
+.green {
+	color: rgb(0, 255, 0);
+}
+
+.red {
+	color: rgb(255, 0, 0);
```
Comment (DanySK) [inline assets/custom-theme.scss:215]:
```suggestion
	color: red;
```

File: assets/custom-theme.scss [hunk 2]
```diff
@@ -207,3 +207,23 @@ kbd {
   font-weight: normal;
 }
 
+.green {
+	color: rgb(0, 255, 0);
```
Comment (DanySK) [inline assets/custom-theme.scss:211]:
```suggestion
	color: green;
```

**Review by DanySK — CHANGES_REQUESTED** [review]
two minimal changes

File: assets/custom-theme.scss (hunk 2 above)
Comment (anitvam) [reply assets/custom-theme.scss:211]: `green` has a different color than `rgb(0, 255, 0);`: it has value `rgb(0, 128, 0);`. 
Do you prefer it?

Comment (DanySK) [reply assets/custom-theme.scss:211]: Didn't know. No then it's fine

**Review by DanySK — APPROVED** [review]

**Pull request merged**